run_service.sh

```diff
@@ -300,7 +300,7 @@ create_storage() {
     # Generate the RPC file
     echo -n "$rpc" > "../$rpc_path"
 
-    # Generate the operator's key
+    # Generate the owner/operator's key
```

Comment:
Why are we adding "owner" everywhere now?

Reply:
Because we use a single address to act as the service owner and agent operator.

Reply:
Ok